java/java.api.common/src/org/netbeans/modules/java/api/common/resources/File.java.template

```diff
@@ -0,0 +1,18 @@
+<#assign licenseFirst = "/*">
```

Comment:
Template files, that are processes by a markup processor that supports comments can carry the license header. So please add it here:

```
<#--

    Licensed to the Apache Software Foundation (ASF) under one
    or more contributor license agreements.  See the NOTICE file
    distributed with this work for additional information
    regarding copyright ownership.  The ASF licenses this file
    to you under the Apache License, Version 2.0 (the
    "License"); you may not use this file except in compliance
    with the License.  You may obtain a copy of the License at

      https://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing,
    software distributed under the License is distributed on an
    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    KIND, either express or implied.  See the License for the
    specific language governing permissions and limitations
    under the License.

-->
```

As this is a freemarker comment, it will not make into the generated document.